Translating Community Page

SanjaySinghRajpoot opened the pull request:
An outcome of Google Summer of Code 2021
Author- Sanjay Singh Rajpoot @SanjaySinghRajpoot
Mentor - Aditya Rana @ranaaditya

Commit:
Translating Community Page

ranaaditya commented on src/data/hi.yml:
```diff
@@ -453,24 +453,38 @@ libraries:
+  community-statement3: >-
+     हमें ये हैशटैग पसंद हैं: #noCodeSnobs (क्योंकि हम समुदाय को महत्व देते हैं
+     दक्षता), #newKidLove (क्योंकि हम सभी ने कहीं न कहीं शुरुआत की है), #unassumeCore
+     (क्योंकि हम ज्ञान को ग्रहण नहीं करते हैं), और #BlackLivesMatter (क्योंकि .)
```
For #blivesmatter tagg please check how to write it in a polite and gentle way

ranaaditya commented on src/data/hi.yml:
```diff
@@ -479,32 +493,32 @@ community:
   develop5: "अपना स्वयं का लाइब्रेरी बनाएं।"
   document-title: "दस्तावेज़।"
-  document1: " दस्तावेज़ीकरण का शौक सभी को होता है। "
+  document1: " सभी को दस्तावेज़ीकरण पसंद है। मदद की जरूरत है "
```
If this is related to docs or documentation please find a better word, I think there should be something better than "daataavez"

SanjaySinghRajpoot replied:
प्रलेखन is  one alternate word for documentation other than "daataavez". So should I change it to प्रलेखन

ranaaditya commented on src/data/hi.yml:
```diff
@@ -479,32 +493,32 @@ community:
-  contributors-conference2: " कार्नेगी मेलन विश्वविद्यालय  में योगदानकर्ताओं के दो सम्मेलन आयोजित किए। कलाकार, डिजाइनर, डेवलपर्स, शिक्षक p5.js परियोजना को आगे बढ़ाने के लिए एक साथ शामिल हुए। "
+  donate3: " को दान के माध्यम से p5.js के विकास में सहायता करें। "
+  contributors-conference-title: "p5.js योगदानकर्ता सम्मेलन"
+  contributors-conference1: "जबकि अधिकांश कार्य ऑनलाइन होते है, हम IRL भी आयोजित करते हैं। हमने पिट्सबर्ग, पीए में "
```
For Pittsburgh, PA you can write full form of PA.
Actually it's  less common practice to abbreviate places name in hindi.

SanjaySinghRajpoot replied:
ok I will make it to Pittsburgh, Pennsylvania

ranaaditya replied:
Better

SanjaySinghRajpoot committed:
Typo error fixed for Hindi language 

black lives matter tag line added
Documentation keyword changed
PA to full form
